frontend/src/Commons/Header/Header.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState, useEffect } from 'react';
+import { screenSizes } from '../../Constants/ScreenSize';
+import { StyledHeader, Logo, Title } from './Header.style';
+
+const Header = () => {
+  const [isDesktop, setIsDesktop] = useState<boolean>(false);
+  useEffect(() => {
+    const handleResize = () => {
+      setIsDesktop(window.innerWidth >= parseInt(screenSizes.desktop, 10));
+    };
```

Comment:
Is there a way to check for this in its parent `App.tsx`? I guess other components will need this in the future

Reply:
Also without this function inside the useEffect() 
```ts
const handleResize = () => {
      setIsDesktop(window.innerWidth >= parseInt(screenSizes.desktop, 10));
    };

    // Initial check
    handleResize();

    // Change Logo based on state
    window.addEventListener('resize', handleResize);

    // Logo changes and removes what's mounted
    return () => {
      window.removeEventListener('resize', handleResize);
    };
```
The logo will not change and will be on top of each other. Because line 22 is only mounted once, so whatever your logo is when the server started or reload, that's what you will have for the rest, but the state still changes so you get another logo. 😆

Reply:
But can't you set the image in css instead of using this? This feels a bit complicated... you're the cook though :smile:

Reply:
I've tried that, it doesn't work for me. If you can make it work, you'll be my saviour 😆